configman/config_manager.py

```diff
@@ -461,7 +461,10 @@ def write_conf(self, config_file_type, opener, skip_keys=None):
                 an_option = option_defs[a_key]
                 if ((not a_key.startswith('admin'))
                     and isinstance(an_option, Option)
-                    and an_option.secret
+                    and (
+                        an_option.secret
+                        or 'password' in an_option.name.lower()
+                    )
```

Comment:
how about if we extend this implicit protection to options with the exactly the name "password" only.  that way we mimimize what we respond to implicitly and don't catch these other cases.